app/packages/feature-flags/src/hooks/useFeature.ts

```diff
@@ -0,0 +1,35 @@
+import { useFeatureCache } from "./useFeatureCache";
+import { FeatureFlag } from "../client";
+import { useTrackEvent } from "@fiftyone/analytics";
+import { useEffect } from "react";
+
+/**
+ * Hook which provides the status of a given feature.
+ *
+ * @param feature Feature identifier
+ * @param enableTracking If enabled, will emit tracking events
+ * @returns true if the feature is enabled, else false
+ */
+export const useFeature = ({
+  feature,
+  enableTracking = true,
```

Comment:
Not a blocker, but my instinct is we will want the default to be `false`. Tracking is really only interesting when we are doing a deliberate test, and most of the time, my bet is that we will not be testing, but deliberately hiding.